CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### 🛡 Security
 - [CVE-2022-37601][CVE-2022-37599] Bump loader-utils to 2.0.4 ([#3318](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3318))
 - [CVE-2022-25860] Bumps simple-git from 3.15.1 to 3.16.0 ([#3345](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3345))
+- [CVE-2020-36632] Bumps flat from 4.1.1 to 5.0.2 ([#3419](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3419))
```

Comment:
```suggestion
- [CVE-2020-36632] [REQUIRES PLUGIN VALIDATION] Bumps flat from 4.1.1 to 5.0.2 ([#3419](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3419)). To the best of our knowledge, this is a non-breaking change, but if your plugin relies on `mocha` tests, validate that they still work correctly (and plan to migrate them to `jest` [in preparation for `mocha` deprecation](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1572). 
```

Reply:
Just to confirm have all mocha tests been removed from both `main` (targeting 3.0) and `2.x` (targeting 2.6) ?

Reply:
And for CHANGELOG didn't realize we can now add call-outs ~ feels like more like a release note thing but it looks good to me

Reply:
> Just to confirm have all mocha tests been removed from both main (targeting 3.0) and 2.x (targeting 2.6) ?

Yep! https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1678

> And for CHANGELOG didn't realize we can now add call-outs ~ feels like more like a release note thing but it looks good to me

Yeah, the idea is for the CHANGELOG to be the initial draft of the release notes. So this is just my suggestion of how we might go about flagging such a callout 🤷